lithops/standalone/standalone.py

```diff
@@ -223,9 +232,15 @@ def run_job(self, job_payload):
                     .format(executor_id, job_id))
         logger.info("View execution logs at {}".format(log_file))
 
-        url = "http://{}:{}/run".format(self.ip_address, PROXY_SERVICE_PORT)
-        r = requests.post(url, data=json.dumps(job_payload))
-        response = r.json()
+        if self.is_lithops_worker:
+            url = "http://{}:{}/run".format('127.0.0.1', PROXY_SERVICE_PORT)
+            r = requests.post(url, data=json.dumps(job_payload), verify=True)
+            response = r.json()
+        else:
+            cmd = ('curl -X POST http://127.0.0.1:8080/run -d \'{}\' '
+                   '-H \'Content-Type: application/json\''.format(json.dumps(job_payload)))
```

Comment:
@JosepSampe BTW, Python has a built-in function [`shlex.quote`](https://docs.python.org/3/library/shlex.html) for escaping a string to make it shell-safe. It may be worthwhile using it everywhere that these shell commands are constructed, just to prevent possible issues. E.g. if somebody's Lithops config contains a value with a `'` character, it will cause the command-line argument in this command to be split into multiple arguments, and curl will complain about invalid JSON.